docs/modules/user-guide/partials/proc_devfile-properties.adoc

```diff
@@ -0,0 +1,258 @@
+[id="proc_devfile-properties_{context}"]
+= Devfile properties 
+
+[role="_abstract"]
+Use this page to work with yaml files of devfile properties. See the following table of contents to go to the devfile property you want to work with:
+
+== SchemaVersion
+
+[source,yaml]
+----
```

Comment:
There is zero explanation or context in this whole guide, which makes it a bit difficult to understand for me right now, even looking at https://odo.dev/file-reference/#devfile-properties. What does the code block do there and what does the table mean? What can users do with this information? Some explanation and context would be useful in my opinion.

Reply:
The table is supposed to help users get started with devfiles with only a quick read. Hence, the bare bones of yaml and properties. However, I can add to the abstract and possibly an intro sentence to communicate this message to users.